### PR TITLE
Allow specifying headers level

### PR DESCRIPTION
**Pull request by SemenchenkoVitaliy**
Based on https://github.com/metarhia/metasync/pull/400#pullrequestreview-227691025.

~~This will allow methods header level to be the same as classes and top-level functions.~~
Headers level for interface name, top-level classes, class methods, class static methods, properties can be specified individually via cli or in the config file. The default value is based on `minHeaderLevel` option.

/cc @lundibundi @belochub @nechaido

### CODE REVIEW

File: lib/generator.js [hunk 1]
```diff
@@ -262,12 +262,13 @@ const checkArgsSignature = (argsSignature, args) => {
 //     customLinks - <Array[]>, custom types links
 // Returns: <string>, md document
 const generateMd = (inventory, options = {}) => {
+  const level = options.minHeaderLevel;
+  const fnLevel = options.increaseFunctionLevel ? level + 2 : level + 1;
```
Comment (nechaido) [inline lib/generator.js]: Maybe:
```suggestion
  const fnLevel = level + 1 + options.increaseFunctionLevel;
```

**Comment by nechaido** [comment]
Please rename PR accordingly.

File: bin/cli.js [hunk 2]
```diff
@@ -84,6 +108,24 @@ const loadFiles = files => {
   return result;
 };
 
+const select = (...args) => {
+  for (const arg of args) {
```
Comment (lundibundi) [inline bin/cli.js]: `return args.find(a => a !== undefined)`?

**Comment by lundibundi** [comment]
Landed in https://github.com/metarhia/metadoc/commit/5c4be79f541067536d68514f1526c296cc77c76c.

**Pull request closed without merging**